advanced/Templates/gravity.db.sql

```diff
@@ -1,11 +1,27 @@
+PRAGMA FOREIGN_KEYS=ON;
+
+CREATE TABLE domain_groups
+(
+	"id" INTEGER PRIMARY KEY AUTOINCREMENT,
+	"enabled" BOOLEAN NOT NULL DEFAULT 1,
+	"description" TEXT
+);
+INSERT INTO domain_groups ("id","description") VALUES (0,'Standard group');
+CREATE TRIGGER domain_groups_standard_group AFTER DELETE ON domain_groups WHEN OLD.id = 0
+    BEGIN
+      INSERT INTO domain_groups ("id","description") VALUES (0,'Standard group');
+    END;
```

Comment:
I rethought the implementation and will remove this altogether. It makes more sense to allow a list to be not managed by any group (it can still be enabled/disabled manually in this case) then to enforce a group association.